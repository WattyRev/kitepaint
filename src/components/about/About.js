import React from "react";
import styled from "styled-components";
import { H1, Spacer, P } from "../../theme";

const StyleWrapper = styled.div`
  max-width: 1280px;
  padding: 16px;
  margin: 0 auto;
`;

const About = () => (
  <StyleWrapper>
    <H1>About</H1>
    <Spacer bottom="md" />
    <P>
      I started creating KitePaint.com in 2014 as a pet project to experiment
      with full stack development. At the time, I built it using AngularJS, but
      I started rebuilding the front end in React in 2018. This site is an
      experiment and a learning experience. While I am experimenting with these
      technologies, I figured that I might as well make something useful.
    </P>
    <Spacer bottom="sm" />
    <P>
      I created a colorizer application for{" "}
      <a href="http://revkites.com" rel="noopener noreferrer" target="_blank">
        Revolution Kites
      </a>{" "}
      some time ago that serves as the initial inspiration for this site. I
      wanted to expand on what I had made for Revolution, but make it bigger,
      more fun, and more accessable for more people. I wanted all kite
      manufactuers to have the ability to get their own colorizers for a
      reasonable price. I wanted to make the custom kite ordering process easier
      for consumers. So, this site is the result. It is built around the concept
      of saving and sharing custom kite designs.
    </P>
    <Spacer bottom="sm" />
    <P>
      For more information, view the project on{" "}
      <a
        href="https://github.com/WattyRev/static.kitepaint"
        target="_blank"
        rel="noopener noreferrer"
      >
        GitHub
      </a>
      .
    </P>
    <Spacer bottom="sm" />
    <P>
      Thanks for checking out KitePaint.com!
      <br />- Spencer &quot;Watty&quot; Watson
    </P>
  </StyleWrapper>
);

export default About;
