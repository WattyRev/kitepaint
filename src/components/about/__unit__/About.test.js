import React from "react";
import { shallow } from "enzyme";
import About from "../About";

describe("About", () => {
  it("renders", () => {
    expect.assertions(1);
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
