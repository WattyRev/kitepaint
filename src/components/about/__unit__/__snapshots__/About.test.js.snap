// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders 1`] = `
<styled.div>
  <Styled(H1)>
    About
  </Styled(H1)>
  <Spacer
    bottom="md"
  />
  <P>
    I started creating KitePaint.com in 2014 as a pet project to experiment with full stack development. At the time, I built it using AngularJS, but I started rebuilding the front end in React in 2018. This site is an experiment and a learning experience. While I am experimenting with these technologies, I figured that I might as well make something useful, and maybe even a little profitable.
  </P>
  <Spacer
    bottom="sm"
  />
  <P>
    I created a colorizer application for
     
    <a
      href="http://revkites.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      Revolution Kites
    </a>
     
    some time ago that serves as the initial inspiration for this site. I wanted to expand on what I had made for Revolution, but make it bigger, more fun, and more accessable for more people. I wanted all kite manufactuers to have the ability to get their own colorizers for a reasonable price. I wanted to make the custom kite ordering process easier for consumers. So, this site is the result. It is built around the concept of saving and sharing custom kite designs.
  </P>
  <Spacer
    bottom="sm"
  />
  <P>
    For more information, view the project on
     
    <a
      href="https://github.com/WattyRev/static.kitepaint"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub
    </a>
    .
  </P>
  <Spacer
    bottom="sm"
  />
  <P>
    Thanks for checking out KitePaint.com!
    <br />
    - Spencer "Watty" Watson
  </P>
</styled.div>
`;
